<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArithmDialog</class>
 <widget class="QDialog" name="ArithmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listOpnum1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>200</width>
     <height>380</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>操作图像1</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="listOpnum2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>200</width>
     <height>380</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>操作图像2</string>
    </property>
    <property name="flags">
     <set>ItemIsEnabled</set>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>计算操作</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboxArithmOpt">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>40</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDo">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>350</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>70</y>
     <width>121</width>
     <height>155</height>
    </rect>
   </property>
   <property name="title">
    <string>计算设定</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkDoRescale">
       <property name="text">
        <string>对图像2ReScale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkDoCenterAligned">
       <property name="text">
        <string>两图像中心对齐</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkDoUseMod">
       <property name="text">
        <string>使用Mod处理
溢出值</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
